cmd/containerboot/main.go

```diff
@@ -310,6 +314,15 @@ authLoop:
 					}
 				}
 			}
+			// TODO (irbekrm): check if destination IP changes (i.e user made a typo & later corrected)
```

Comment:
That will require a container restart to register atm

Reply:
I'll think about that separately

Reply:
This is now solved by the last-target-ip annotation that forces pod restart, so I've removed the comment